pandas/tests/extension/base/interface.py

```diff
@@ -75,3 +75,15 @@ def test_copy(self, data):
 
         data[1] = data[0]
         assert result[1] != result[0]
+
+    def test_view(self, data):
+        # view with no dtype should return a shallow copy, *not* the same
+        #  object
```

Comment:
do you need to also test with a dtype != None? (e.g. that this raises NIE)

Reply:
I guess to make sure the kwarg is accepted, sure